packages/button/src/Button.tsx

```diff
@@ -107,13 +107,23 @@ const Button: ExtendableBox<
     type = 'button';
   }
 
+  // If the link is disabled, render as a div
+  const getButtonTag = (
+    isAnchor: boolean,
+    disabled: boolean,
+  ): 'a' | 'button' | 'div' => {
+    if (!isAnchor) return 'button';
+    if (disabled) return 'div';
+    return 'a';
+  };
```

Comment:
Since we can't disable a link with the `disabled` attribute, we have to change the tag type to a `div` to make ensure clicking the button won't navigate

Reply:
Are there any a11y concerns with keeping an `href` attribute on a `div` (and expecting it _not_ to work)?

Reply:
Hmm... since disabled buttons should still be focusable when disabled for accessibility reasons, I think changing this to a div is probably not the best solution.

I've actually seen some examples of rendering a button instead of a link in these cases, and handling `disabled` in the same way that buttons do. That seems like a stronger approach for accessibility reasons.

Reply:
That makes a lot of sense. Putting an `href` attribute on a `button` is still strange, but no more strange than on a `div`